hifiberrydsp/alsa/alsasync.py

```diff
@@ -72,7 +72,7 @@ def __init__(self):
         self.dsp = Adau145x
         self.volume_register_length = self.dsp.WORD_LENGTH
         self.finished = False
-        self.pollinterval = 0.1
+        self.pollinterval = 10
```

Comment:
Please change to 1 second

Reply:
Alternatively add a command line option to set the interval

Reply:
1 second (now in milliseconds) is fine with me.